src/test/java/com/tesobe/obp/user/EntitlementsTest.java

```diff
@@ -15,7 +15,14 @@
 
     @Test
     public void entitlementsUser() throws Exception {
-        String entz = entitlementsApiClient.getEntitlements(user);
-        Assert.assertNotNull(entz);
+        try {
```

Comment:
in the junit test method, should not try catch the exception, just throw the exception.
only if we want check the exception detail, take message as example, we need catch it and assert the message is equals some value.